source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -31,12 +31,36 @@ step before it considers the connection string and SDAM specifications. In this
 host names is replaced with a single host name. The format is:
 
 ```
-mongodb+srv://{hostname}.{domainname}/{options}
+mongodb+srv://{hostname}/{options} 
 ```
 
 `{options}` refers to the optional elements from the [Connection String](../connection-string/connection-string-spec.md)
 specification following the `Host Information`. This includes the `Auth database` and `Connection Options`.
 
+For the purposes of this document, `{hostname}` will be divided using the following terminology. If an SRV `{hostname}`
+has:
+
+1. Three or more `.` separated parts, then the left-most part is the `{subdomain}` and the remaining portion is the
```

Comment:
`subdomain` is defined here but I don't see it ever used in the rest of the spec. What is its purpose?

Reply:
`{subdomain}` is necessary distinction for SRVs with 3+ parts since with these SRVs, we want a less strict domain match.

For example,  the hostname returned by the SRV record `blogs.mongodb.com`  should share the `{domainname}='mongodb.com`.  The SRV record `mongo.local` has only 2 parts and no hostname, so the hostname returned by the SRV should match the entire domain (ex: `cluster_1.mongo.local`).

Reply:
> mongo.local has only 2 parts and no hostname

Now I'm more confused. Isn't "hostname" defined as the whole thing: `mongodb+srv://{hostname}/{options}`? 

Also, I get that `{subdomain}` is defined here but I still don't see it ever referenced in the rest of the spec which is confusing to me. Is this accurate to say:

> Only `{hostname} is used during SRV record verification and ``{subdomain}` is ignored.

Reply:
Bookkeeping: Resolved over slack, added in line clarifying subdomain's role / lack thereof in the verification.